cloudevents/formats/avro-turbo-format.md

```diff
@@ -0,0 +1,109 @@
+# Avro Turbo Event Format for CloudEvents - Version 1.0.3-wip
+
+## Abstract
+
+The Avro Turbo Format for CloudEvents defines how events are expressed in
```

Comment:
While I don't love how both Avro formats sound definitive as if they're the only format, we do the same for Protobuf so I'm not suggesting an actual change here.